sourmash_lib/kmer_min_hash.hh

```diff
@@ -254,14 +273,22 @@ class KmerMinAbundance: public KmerMinHash {
     KmerMinAbundance(unsigned int n, unsigned int k, bool prot) :
         KmerMinHash(n, k, prot) { };
 
+    KmerMinAbundance(unsigned int n, unsigned int k, bool prot,
+                     HashIntoType mx) :
+        KmerMinHash(n, k, prot, mx) { };
+
     virtual void add_hash(HashIntoType h) {
-        if (mins.size() < num) {
+        if (max_hash && h > max_hash) {
+            return;
+        }
+
+        if (!num || mins.size() < num) {
```

Comment:
Quick comment: since we are using `C++11`, we can also use `or` instead of `||` and `and` instead of `&&` =]